### DISCUSSION TITLE
Uncompiled version of RTL SCSS for customizing Bootstrap

### DISCUSSION THREAD

**Discussion by sinjuangajan** [twbs/bootstrap#39638] [Q&A]
Where can I get the original RTL SCSS code for Bootstrap without it being compiled?  I'm using a Sass compiler for my theme development, and converting it to RTL is proving to be a challenge. Any advice on how to customize Bootstrap's SCSS for RTL layouts would be helpful. Thank you!

I'm having trouble customizing the minified version to suit my needs, especially when working with utility classes.

**Comment by julien-deramond** [comment]
I've transformed your issue into a discussion @sinjuangajan.

> Where can I get the original RTL SCSS code for Bootstrap without it being compiled?

The SCSS code is actually the same as the one you can find in `node_modules/bootstrap/scss`. The RTL differences are present in the files with comments like for instance `/* rtl:begin:ignore */ .... /* rtl:end:ignore */`thanks to [RTLCSS](https://rtlcss.com).

Additional information can be found at https://getbootstrap.com/docs/5.3/getting-started/rtl/.

**Reply by sinjuangajan** [reply]
@julien-deramond Thank you for your response. Could you update the RTL documentation for RTLCSS to be similar to the SASS documentation? It would be a significant help for developers.

**Comment by sinjuangajan** [comment]
I suggest including the following documentation in the RTLCSS project:

RTLCSS Integration and Usage Guide
Project Structure
Setting Up Your Environment
Working with Your Source Files
Compiling and Generating RTL Styles
Required HTML for RTL Support
Including RTL CSS

**Comment by sinjuangajan** [comment]
I have prepared a sample documentation. Could you please check it at

[RTL Documentation Sample](http://sinju.in/labs/bootstrap/rtl-documentation/)

**Reply by julien-deramond** [reply]
Thanks for this work @sinjuangajan. I'll check that as soon as I can.
